benchmark/schema/operator-distributed.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "operator-distributed.json",
+  "title": "Distributed Operator",
+  "description": "Reusable distributed operator definition",
+  "type": "object",
+  "properties": {
+    "filename": {
+      "type": "string"
+    },
+    "stencil": {
+      "type": "object",
+      "properties": {
+        "name": {
```

Comment:
```suggestion
        "discretization": {
```

Reply:
IMO name is shorter and just as descriptive (with the right documentation). I will leave it as is.

Reply:
That's better, thanks for the suggestion!